src/aws_credentials.erl

```diff
@@ -8,7 +8,9 @@
 -behaviour(gen_server).
 
 %% As per
-%% http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#instance-metadata-security-credentials
+%% http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/
+%% iam-roles-for-amazon-ec2.html#instance-metadata-security-credentials
+%%
```

Comment:
URLs should not get whitespace in the middle of them, I should be able to copy and paste them